api/server/router/system/system_routes.go

```diff
@@ -91,6 +91,12 @@ func (s *systemRouter) getInfo(ctx context.Context, w http.ResponseWriter, r *ht
 				info.OperatingSystem = "<unknown>"
 			}
 		}
+		if versions.LessThan(version, "1.44") {
+			for k, rt := range info.Runtimes {
+				// Status field introduced inl API v1.44.
```

Comment:
Arf; looks like I mad a typo in my PR

I'll push a fix to your branch so that we can move ahead with this one.